tests/unit/type/list.tcl

```diff
@@ -305,83 +307,79 @@ start_server [list overrides [list save ""] ] {
         set s0 [s used_memory]
         assert {$s0 > $str_length}
         assert {[r llen lst] == 3}
-        set s0 [r rpop lst]
+        assert_equal [r rpop lst] "9"
         assert_equal [read_big_bulk {r rpop lst}] $str_length
-        assert {$s0 eq "9"}
         assert {[r llen lst] == 1}
         assert_equal [read_big_bulk {r rpop lst}] $str_length
-   }
+   } {} {large-memory}
```

Comment:
tags are by default executed (unless `--tags -<tagname>` is added, or `--tags <tagname>` is added.
i.e. one means run everything except a certain tag, the other means run only a certain tag.

but now we need that tag to be skipped by default.
so i think we need some explicit mechanism rather than rely on the tags.

I think the mechanism we should follow is the `cluster:skip` tag.
see this code in server.tcl:
```
     if {$::cluster_mode && [lsearch $tags "cluster:skip"] >= 0} {
        set err "Not supported in cluster mode"
        return 0
    }
```

i.e. i think the tag here in list.tcl is ok (it tags a test as needing large memory), no need to change this line.
but we need to do these too:
1. add an optional argument to test_helper.tcl (like `--cluster-mode`)
2. add skipping code in server.tcl (`tags_acceptable`) like the one i pasted above)
3. add a note about it in `tests/README.md` (like the one who mentions `--cluster-mode`)